### PR TITLE
Feature visualize with best parameter

### PR DESCRIPTION
**Pull request by PaulJonasJost**
Serves as a function to conveniently check the model fit i.e. the trajectories of the model given the optimised parameters. Open for any changes.

### CODE REVIEW

File: pypesto/visualize/model_fit.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import petab
```
Comment (yannikschaelte) [inline pypesto/visualize/model_fit.py:1]: short file-level docstring please """thisfileisabout""

File: pypesto/visualize/model_fit.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+from ..result import Result
+from amici.petab_objective import rdatas_to_simulation_df
+
+
+def visualize_optimized_model_fit(petab_problem: petab.Problem,
+                                  result: Union[Result, Sequence[Result]],
+                                  **kwargs
+                                  ):
+    """
+    Function calls the petab.visualize file of the petab_problem and
```
Comment (yannikschaelte) [inline pypesto/visualize/model_fit.py:15]: maybe one-line summary in the first line "Visualize the optimized model fie" or sth like that?

Comment (yannikschaelte) [reply pypesto/visualize/model_fit.py:15]: how can one call a visualize file

File: pypesto/visualize/model_fit.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    visualizes the fit of the optimized parameter. Possible additional
+    argument is `subplot_dir` to specify the directory each subplot is
```
Comment (yannikschaelte) [inline pypesto/visualize/model_fit.py:17]: "further keyword arguments are delegated to ... see there for further information" but yes makes sense to document commonly used arguments like subplot_dir here.

**Review by yannikschaelte — APPROVED** [review]
can you upload an example result plot here to the PR? Just to document what it looks/looked like. Also if possible at dummy test.

**Comment by PaulJonasJost** [comment]
> can you upload an example result plot here to the PR? Just to document what it looks/looked like. Also if possible add dummy test.

What do you mean what it looked like?

will upload an example plot 👍

**Comment by yannikschaelte** [comment]
> > can you upload an example result plot here to the PR? Just to document what it looks/looked like. Also if possible add dummy test.
> 
> What do you mean what it looked like?
> 
> will upload an example plot

I mean if someone breaks it in the future, one can find what it should look like

**Comment by PaulJonasJost** [comment]
> > > can you upload an example result plot here to the PR? Just to document what it looks/looked like. Also if possible add dummy test.
> > 
> > 
> > What do you mean what it looked like?
> > will upload an example plot
> 
> I mean if someone breaks it in the future, one can find what it should look like

added a use case in the petab_import notebook. Not sure whether uploading an example plot is that useful as it should really look just like the petab visualisation, but instead of the petab parameters, it uses the optimised parameter.

File: pypesto/visualize/model_fit.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    Function calls the PEtab visualization file of the petab_problem and
+    visualizes the fit of the optimized parameter. Common additional
+    argument is `subplot_dir` to specify the directory each subplot is
+    saved to. Further keyword arguments are delegated to
+    petab.visualize.plot_with_vis_spec, see there for more information.
+
+    Parameters
+    ----------
+    petab_problem:
+        The petab_problem that was optimized.
```
Comment (dweindl) [inline pypesto/visualize/model_fit.py:29]:
```suggestion
        The :py:class:`petab.Problem` that was optimized.
```

**Review by dweindl — APPROVED** [review]

**Pull request merged**